server/src/main/java/server/haengdong/domain/event/Event.java

```diff
@@ -51,4 +51,8 @@ private void validateName(String name) {
     private boolean isBlankContinuous(String name) {
         return name.contains(SPACES);
     }
+
+    public boolean isNotValidToken(String token) {
```

Comment:
c: 메서드 이름을 다음과 같이 변경해 보는 것은 어떨까요? valid라는 표현 대신 mismatch를 사용해 일치 여부를 확인한다는 것을 명시하면 더 의도가 잘 드러날 것 같습니다.

```suggestion
    public boolean isTokenMismatch(String token) {
```